tests/test-validate-app-receipt.js

```diff
@@ -75,6 +75,63 @@ describe('fxpay.validateAppReceipt()', function() {
     validator.finish();
   });
 
+  it('fails when receipt was issued by a disallowed store', function(done) {
+    // Disallow all other stores:
+    helper.appSelf.manifest.installs_allowed_from = [
```

Comment:
Yeah, it gets set before each test here https://github.com/mozilla/fxpay/pull/75/files#diff-d5ebdbfb88cc7626c6e2f1d9f32e6d5aR161